src/App.tsx

```diff
@@ -15,48 +16,119 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphabet = 'Sort alphabetically',
+  Length = 'Sort by length',
+}
+
+type Button = {
+  name: SortType | 'Reverse',
+  color: 'is-info' | 'is-success' | 'is-warning',
+};
+
+const buttons: Button[] = [
+  {
+    name: SortType.Alphabet,
+    color: 'is-info',
+  },
+  {
+    name: SortType.Length,
+    color: 'is-success',
+  },
+  {
+    name: 'Reverse',
+    color: 'is-warning',
+  },
+];
+
+let goods = [...goodsFromServer];
+
+function operateGoods(stateSort: string | null, stateReverse: string | null) {
```

Comment:
better to init `goods` variable in `operateGoods`
```suggestion
function operateGoods(stateSort: string | null, stateReverse: string | null) {
  let goods = [...goodsFromServer];
```

Reply:
[comment](https://github.com/mate-academy/react_list-of-goods-hooks/pull/1139/files#r1263490865) not fixed